packages/components/src/internal/components/labels/LabelsConfigurationPanel.tsx

```diff
@@ -345,12 +346,13 @@ export const LabelTemplateDetails: FC<LabelTemplateDetailsProps> = memo(props =>
 
 export const LabelsConfigurationPanel: FC<LabelTemplatesPanelProps> = memo(props => {
     const { api, setIsDirty, defaultLabel } = props;
-    const { user } = useServerContext();
+    const { user, container } = useServerContext();
     const [templates, setTemplates] = useState<LabelTemplate[]>([]);
     const [error, setError] = useState<string>();
     const [selected, setSelected] = useState<number>();
     const [newDefaultLabel, setNewDefaultLabel] = useState<number>(defaultLabel);
     const addNew = useMemo(() => selected === NEW_LABEL_INDEX, [selected]);
+    const showAdd = container.isProject || !isProductProjectsEnabled();
```

Comment:
I ran across a similar scenario recently when fixing issue 47502. In that case, I used the `isAppHomeFolder` helper (https://github.com/LabKey/labkey-ui-components/pull/1156). Should this do the same?